src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceImplTest {
+    private static FileWriterService fileWriterService;
+
+    @BeforeAll
+    static void beforeAll() {
+        fileWriterService = new FileWriterServiceImpl();
+    }
+
+    @Test
+    void write_validPath_ok() {
```

Comment:
too large method. Is it possible to split it into smaller ones / use additional utility methods